packages/s/sol2/xmake.lua

```diff
@@ -28,8 +29,7 @@ package("sol2")
     end)
 
     on_install(function (package)
-        local configs = {}
-        import("package.tools.cmake").install(package, configs)
+        import("package.tools.cmake").install(package, {})
```

Comment:
remove `, {})`

Reply:
`import("package.tools.cmake").install(package)`

Reply:
changed in commit `exclude iphoneos attempt 2`